pkg/crc/constants/constants.go

```diff
@@ -29,7 +29,8 @@ const (
 	CrcLandingPageURL    = "https://cloud.redhat.com/openshift/install/crc/installer-provisioned" // #nosec G101
 	PullSecretFile       = "pullsecret.json"
 
-	DefaultOcUrlBase = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest"
+	DefaultOcUrlBase   = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest"
+	CrcTrayDownloadURL = "https://github.com/anjannath/crc-macos-tray/releases/download/%s/crc_tray_1.0.0-alpha.tar.gz"
```

Comment:
This URL will have to changed, once we move the tray repo to code-ready namespace

Reply:
@anjannath Since the repo is already migrated can you change this URI now also should this be part of this contants file or be part of macos specific constant file?

Reply:
So it has to be in the constants file, since the `crc-embedder` should be able to find even if we build the embedded binary in a non macOS environment. Its based on teuf's patches for embedding oc and other assets.